### ISSUE TITLE
German QWERTZ layout missing (option) to constantly show Umlaute (äöüß)

### ISSUE THREAD

**Issue by ctshh** [rkkr/simple-keyboard#254]
The [german QWERTZ keyboard shows the Umlaute](https://en.wikipedia.org/wiki/German_keyboard_layout) ÄÖÜß/äöüß constantly*.
The "Deutsch (QWERTZ)" setting should also reflect this ([example from iOS](https://forums.macrumors.com/threads/german-character-ss-ess-tset-on-iphone-keyboard.650489/)).

_*) [A capital ß (ẞ) does exist](https://en.wikipedia.org/wiki/%C3%9F), yet this is not reflected on any keyboard I know nor is it used anywhere, lacking words that start with a capital ß._

(And, of course: thanks for this great piece of unobtrusive, nicely working piece of software! :)

**Comment by Sogolumbo** [comment]
Yes. The german layout in this program is wrong. I think the swiss layout is pretty much what we need for the german layout.
The wikipedia page shows how the ö ä ü keys are part of the standard layout.

![grafik](https://user-images.githubusercontent.com/33571916/162207954-af684111-5d22-44a3-889f-220e5147cb7b.png)
(Tastaturbelegung T1)
https://de.wikipedia.org/wiki/Tastaturbelegung#Deutschland

**Comment by Sogolumbo** [comment]
The capital letter for `ß` is `ẞ (U+1E9E)` (not `SS`, even if it's often used that way because the capital ß is pretty new and not widely supported yet).

It seems to me like the java string library is wrong here:
https://bugs.openjdk.java.net/browse/JDK-4901305
